src/todo.js

```diff
@@ -51,51 +44,67 @@ export default class Todo {
         </div>
        
         `;
+  });
+
+  const deleteBtn = document.querySelectorAll('#delete');
+  const textLabel = document.querySelectorAll('#description');
+  const dropdownBtn = document.querySelectorAll('#dropdown');
+  const divDropdownBtn = document.querySelectorAll('#sectiontohide');
+  const checkBoxes = document.querySelectorAll('#todoCheck');
+
+  // eslint-disable-next-line
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.